### ISSUE TITLE
钢铁比赛复现

### ISSUE THREAD

**Issue by MiaoRain** [MiaoRain/lund#11]
Dice loss / Focal loss
https://www.kaggle.com/bigironsphere/loss-function-library-keras-pytorch
